.github/actions/test-package/action.yml

```diff
@@ -55,32 +77,33 @@ runs:
       set -euxo pipefail
       workspace=$(pwd)
       cd k-${BASE_DISTRO}
-      docker run                                           \
-        --name k-package-test-${BASE_DISTRO}-${GITHUB_SHA} \
-        --rm -it                                           \
-        --detach                                           \
-        --workdir /opt/workspace                           \
-        -v "${workspace}:/opt/workspace"                   \
-        -v "/etc/passwd:/etc/passwd:ro"                    \
-        -v "/etc/group:/etc/group:ro"                      \
+      docker run                                                      \
+        --name k-package-test-${BASE_OS}-${BASE_DISTRO}-${GITHUB_SHA} \
+        --rm -it                                                      \
+        --detach                                                      \
+        --workdir /opt/workspace                                      \
+        -v "${workspace}:/opt/workspace"                              \
+        -v "/etc/passwd:/etc/passwd:ro"                               \
+        -v "/etc/group:/etc/group:ro"                                 \
         ${BASE_OS}:${BASE_DISTRO}
-  - name: 'Test Package: ${{ inputs.distro }}'
+
+  - name: 'Test Package: ${{ inputs.os }}:${{ inputs.distro }}'
     shell: bash {0}
     env:
+      BASE_OS: ${{ inputs.os }}
       BASE_DISTRO: ${{ inputs.distro }}
+      TEST_PACKAGE: ${{ inputs.test-package }}
+      PKG_NAME: ${{ inputs.pkg-name }}
     run: |
       set -euxo pipefail
-      docker exec -t k-package-test-${BASE_DISTRO}-${GITHUB_SHA} /bin/bash -c "k-${BASE_DISTRO}/src/main/scripts/test-in-container-debian"
+      docker exec -t k-package-test-${BASE_OS}-${BASE_DISTRO}-${GITHUB_SHA} /bin/bash -c "${TEST_PACKAGE} ./${PKG_NAME}"
```

Comment:
Then here we call `TEST_PACKAGE`, which we tell where the package has been moved to.